configmap-reload.go

```diff
@@ -153,13 +354,15 @@ func setFailureMetrics(h, reason string) {
 	lastReloadError.WithLabelValues(h).Set(1.0)
 }
 
-func setSuccessMetrict(h string, begun time.Time) {
+func setSuccessMetrics(h string, begun time.Time) {
 	requestDuration.WithLabelValues(h).Set(time.Since(begun).Seconds())
 	successReloads.WithLabelValues(h).Inc()
 	lastReloadError.WithLabelValues(h).Set(0.0)
 }
 
 func isValidEvent(event fsnotify.Event) bool {
+	// for testing, have this return true
```

Comment:
probably don't need this anymore?

Reply:
I think this is useful for people who want to contribute to this in the future.